<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Salesman</class>
 <widget class="QWidget" name="Salesman">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="LE_name">
       <property name="placeholderText">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_fees">
       <property name="placeholderText">
        <string>Комиссионные</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_sold_item">
       <property name="placeholderText">
        <string>Проданная позиция</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_sold_amount">
       <property name="placeholderText">
        <string>Кол-во</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_sale_price">
       <property name="placeholderText">
        <string>Цена продажи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_date">
       <property name="placeholderText">
        <string>Дата продажи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_add">
       <property name="text">
        <string>Добавить в базу</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="LE_del_name">
       <property name="placeholderText">
        <string>ФИО</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_del_item">
       <property name="placeholderText">
        <string>Проданная позиция</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_del_amount">
       <property name="placeholderText">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="LE_del_date">
       <property name="placeholderText">
        <string>Дата продажи</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PB_delete">
       <property name="text">
        <string>Удалить из базы</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="PB_history">
     <property name="text">
      <string>Открыть историю продаж</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
